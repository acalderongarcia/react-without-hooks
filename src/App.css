*{
  font-family: 'Roboto', sans-serif;
}

.App {
  text-align: center;
}

.firstSection{
  background-color:transparent !important;
}

.secondSection{
  background-color:rgb(243, 242, 242) !important;
}

.logo{
  height: 325px;
}

.downloadApple{
  height: 50px;
}

.blue-text{
  color: rgb(28, 69, 255);
}

.btn{
  border-radius: 0!important;
}

.btn-blue{
  background-color: #1976d2!important;
  color: #fff!important;
}
