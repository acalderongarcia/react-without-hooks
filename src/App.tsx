import React from 'reactn';
import './App.css';
import 'bootstrap/dist/css/bootstrap.css';
import Header from "./components/Header";
import PropTypes from "prop-types";
import t from "./utils/localization/Localization";

class App extends React.Component{

    static propTypes = {
        children: PropTypes.object.isRequired
    };

    render() {
        t.setLanguage(this.global.currentLanguage)
        const children = this.props;
        return (
            <div>
                <Header/>
                {children.children}

                <body> 
                sssFFFFFFFFFFFFFFFFFF
                
                </body>
            </div>
        );
    }
}

export default App;
