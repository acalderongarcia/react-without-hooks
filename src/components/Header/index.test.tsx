import React from 'reactn';
import renderer from 'react-test-renderer';
import '../../App.css'
import 'bootstrap/dist/css/bootstrap.css';

describe('Testing src/components/Header/.tsx', () => {

  test('Header it renders', () => {
    // Before
    const wrapper = renderer.create(
        <div className="Header">
        </div>
    );
    // Then
    let tree = wrapper.toJSON();
    // After
    expect(tree).toMatchSnapshot();
  });

});
