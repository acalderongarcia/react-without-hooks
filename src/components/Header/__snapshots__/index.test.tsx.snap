// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing src/components/Header/.tsx Header it renders 1`] = `
<div
  className="Header"
/>
`;
